.htaccess untracked on deployment host

From a github actions run:

2s
Run echo APP_HOST: $APP_HOST DEPLOY_DIR: $DEPLOY_DIR
APP_HOST: planttracer.com DEPLOY_DIR: planttracer.com/web-static
Agent pid [1](https://github.com/Plant-Tracer/web-static/actions/runs/11463204234/job/31896499664#step:3:1)699
Identity added: (stdin) (planttracer@patroller)
256 SHA256:xMgQOuWi9KyIZ0uB+VyJmZ3gXRuMgog/uRkazzyg2lU planttracer@patroller (ED25519)
Warning: Permanently added 'planttracer.com' (ED25519) to the list of known hosts.
Python 3.10.12
iad1-shared-e1-33
/home/planttracer
total 8
-rw-r--r-- 1 planttracer pg480539  53 Sep 27  [20](https://github.com/Plant-Tracer/web-static/actions/runs/11463204234/job/31896499664#step:3:21)23 README.md
Already on 'main'
-rw-r--r-- 1 planttracer pg480539 830 Sep 27  2023 pyserver.py
drwxr-xr-x 4 planttracer pg480539 304 Oct 16 18:13 static
Your branch is up to date with 'origin/main'.
From https://github.com/Plant-Tracer/web-static
   534167b..6108734  main       -> origin/main
 * [new branch]      15-add-meta-description-tags -> origin/15-add-meta-description-tags
 * [new tag]         v1.0.1     -> v1.0.1
error: The following untracked working tree files would be overwritten by merge:
Updating 534167b..6108734
	static/.htaccess
Please move or remove them before you merge.
Aborting
Error: Process completed with exit code 1.

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: "Deploy to $APP_HOST $DEPLOY_DIR"
         run: |

